pkg/controller/node/node_controller.go

```diff
@@ -18,6 +17,8 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/qinqon/kube-admission-webhook/pkg/controller"
```

Comment:
Oh wait a second. You are replacing the controller-runtime library with your fork? That can get us to vendoring hell and bring random issues. Is it really necesarry?

Reply:
cert-manager has to run unique at cluster so we need leader election at knmstate to run this controller unique o we use separate Deployement for it (maybe with webhook too) but this will not fix kube-mac-pool issue since it's active active so they need some form of Non Leader Election controller if we don't want to run cert-manager a separate Deployment.